datadog_checks_base/datadog_checks/base/utils/db/statement_metrics.py

```diff
@@ -2,10 +2,18 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import logging
+from typing import Any, Callable, Dict, List, Tuple
 
 logger = logging.getLogger(__name__)
 
 
+# Complex Types
+Row = Dict[str, Any]
+RowKey = Tuple[Any]
```

Comment:
You probably want:

```suggestion
RowKey = Tuple[Any, ...]
```

Or, even simpler:

```suggestion
RowKey = tuple
```